jdaviz/configs/imviz/tests/test_linking.py

```diff
@@ -237,20 +237,96 @@ def test_wcslink_rotated(self):
         assert self.viewer.label_mouseover.value == '+1.00000e+00 '
         assert self.viewer.label_mouseover.world_ra_deg == ''
         assert self.viewer.label_mouseover.world_dec_deg == ''
+        assert not self.viewer.label_mouseover.unreliable_world
 
         self.viewer.on_mouse_or_key_event({'event': 'keydown', 'key': 'b',
                                            'domain': {'x': 0, 'y': 0}})
         assert self.viewer.label_mouseover.pixel == 'x=00.0 y=00.0'
         assert self.viewer.label_mouseover.value == '+1.00000e+00 electron / s'
         assert self.viewer.label_mouseover.world_ra_deg == '3.5817255823'
         assert self.viewer.label_mouseover.world_dec_deg == '-30.3920580740'
+        assert not self.viewer.label_mouseover.unreliable_world
 
         self.viewer.on_mouse_or_key_event({'event': 'keydown', 'key': 'b',
                                            'domain': {'x': 0, 'y': 0}})
         assert self.viewer.label_mouseover.pixel == 'x=02.7 y=09.8'
         assert self.viewer.label_mouseover.value == ''
         assert self.viewer.label_mouseover.world_ra_deg == '3.5817255823'
         assert self.viewer.label_mouseover.world_dec_deg == '-30.3920580740'
+        assert not self.viewer.label_mouseover.unreliable_world
+
+        # Make sure GWCS now can extrapolate. Domain x,y is for FITS WCS data
+        # but they are linked by WCS.
+        self.viewer.on_mouse_or_key_event({'event': 'mousemove',
+                                           'domain': {'x': 11.281551269520731,
+                                                      'y': 2.480347927198246}})
+        assert self.viewer.label_mouseover.pixel == 'x=-1.0 y=-1.0'
+        assert self.viewer.label_mouseover.value == ''
+        assert self.viewer.label_mouseover.world_ra_deg == '3.5815955408'
+        assert self.viewer.label_mouseover.world_dec_deg == '-30.3919405616'
+        # FITS WCS is reference data and has no concept of bounding box.
+        assert not self.viewer.label_mouseover.unreliable_world

```

Comment:
I think this line should fail now because you also check if cursor is outside the GWCS bounding box.